client/src/components/Card.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { animated, interpolate } from "react-spring";
+import Button from '@material-ui/core/Button';
+
+class Card extends React.Component {
```

Comment:
Is there any reason this is a class component, and not functional? If not, let's try to keep things consistent as functional components as you've done with flashcards

Reply:
Will do!

Reply:
Looks good!